contrib/ohi-release-notes/run.sh

```diff
@@ -111,6 +115,12 @@ if ! [ -f "$DICTIONARY" ]; then
     curl -s -o "$DICTIONARY" "$DICTIONARY_URL"
 fi
 
+# fetch default excluded-dependencies-manifest by default
+if ! [ -f "$EXCLUDED_DEPENDENCIES_MANIFEST" ]; then
```

Comment:
If a user uses `--excluded-dependencies-manifest`, at line 95 this variable is not going to be a file so the test `-f` is going to fail and you are going to download the file and overwrite the variable with a new file.